cli_tools/common/image/importer/inflater.go

```diff
@@ -23,6 +25,8 @@ import (
 	"github.com/GoogleCloudPlatform/compute-image-tools/proto/go/pb"
 )
 
+var allowedAPIFormatsEx = regexp.MustCompile("^(vpc)$")
```

Comment:
both vpc and vhd? I am not sure whether vhd is also a possible output, maybe not.

Reply:
I tested the valid outputs using qemu, and it classifies vhd/vpc strictly to `vpc`.